### PR TITLE
address rubocop rspec errors

### PR DESCRIPTION
**Pull request by ndushay**
## Why was this change made? 🤔

Based somewhat on sul-dlss/DeveloperPlaybook/pull/195, address rubocop RSpec errors for more idiomatic test code.

## How was this change tested? 🤨

⚡ ⚠ If this change has cross service impact, ***run [integration tests](https://github.com/sul-dlss/infrastructure-integration-test) that do accessioning and/or create_preassembly_image_spec as it tests full preservation*** and/or test in stage environment, in addition to specs. ⚡

### CODE REVIEW

**Review by peetucket — APPROVED** [review]

File: spec/preservation_ingest/complete_ingest_spec.rb [hunk 1]
```diff
@@ -17,7 +17,7 @@
     end
 
     it 'updates the accessionWF when no errors are raised' do
-      expect(this_robot.workflow_service).to receive(:update_status)
+      allow(this_robot.workflow_service).to receive(:update_status)
```
Comment (mjgiarlo) [inline spec/preservation_ingest/complete_ingest_spec.rb:20]: It's hard to know for some of the `expect` changes in this commit whether they were intended *only* to set a stub or also to verify the stub is called. If we think the former, A-OK. If we think the latter, these changes potentially remove test conditions. What do you think?

Comment (ndushay) [reply spec/preservation_ingest/complete_ingest_spec.rb:20]: I looked at each test and left in `have_received` when I felt the test or the test description warranted.

Comment (ndushay) [reply spec/preservation_ingest/complete_ingest_spec.rb:20]: I looked over my changes again and added 2 more `expect` lines back in, but I am ok with everything else as is.

Comment (mjgiarlo) [reply spec/preservation_ingest/complete_ingest_spec.rb:20]: Works for me, thanks :)

**Review by mjgiarlo — APPROVED** [review]

**Pull request merged**